src/polyswarmd/eth.py

```diff
@@ -113,6 +118,10 @@ def post_transactions():
             logger.exception('Unexpected exception while parsing transaction')
             continue
 
+        if withdrawal_only and not is_withdrawal(tx):
+            errors.append('Invalid transaction tx {0}: only withdrawals allowed without api-key'.format(tx.hash.hex()))
```

Comment:
"Invalid transaction for tx {0}: only withdrawals allowed without an API key" for consistency